src/features/governance/fetchFromRepository.ts

```diff
@@ -154,8 +154,14 @@ async function fetchGithubDirectory(
 
 async function fetchGithubFile(url: string): Promise<string | null> {
   try {
+    const headers = new Headers();
+    if (process.env.NEXT_PUBLIC_GITHUB_PAT) {
+      headers.append('Authorization', process.env.NEXT_PUBLIC_GITHUB_PAT);
+    }
     logger.debug('Fetching github file', url);
-    const response = await fetch(url);
+    const response = await fetch(url, {
+      headers,
+    });
```

Comment:
This resolves 403 rate limit exceeded you can encounter in dev mod when fetching more than 60 times an hour